examples/indentation/indentation_test.go

```diff
@@ -0,0 +1,44 @@
+package main
+
+import (
+	"testing"
+    "io/ioutil"
+	"path/filepath"
+)
+
+func TestIndentation(t *testing.T) {
+	files := testIndentationFiles(t)
+	for _, file := range files {
+		pgot, err := ParseFile(file)
+		if err != nil {
+			t.Errorf("%s: pigeon.ParseFile: %v", file, err)
+			continue
+		}
+        got,err := pgot.(ProgramNode).exec()
+        if err != nil {
+			t.Errorf("%s: ProgramNode.exec: %v", file, err)
+			continue
+		}
+        
+        exp := 42
```

Comment:
On one hand, the test code is prepared to process several files from the `testdata` directory, on the other hand, the expected result is always 42. Maybe this could be optimized, e.g. by adding a `map[filename]result`.

Reply:
ok.